interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -404,7 +404,23 @@ def test_modify_thermo(self):
             else:
                 # Rate should be the same if reaction does not involve OH
                 self.assertAlmostEqual(w2[i] / w1[i], 1.0)
-
+    def test_pdep_err(self):
+        err_msg = ("Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'",
+                    "at P = 32019 , T = 500.0",
+                    "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'",
+                    "at P = 32019 , T = 1000.0",
+                    "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'",
+                    "at P = 1.0132e+05 , T = 500.0",
+                    "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'",
+                    "at P = 1.0132e+05 , T = 1000.0",
+                    "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3 + CO2'",
+                    "at P = 1.0132e+05 , T = 500.0")
```

Comment:
You can delete the repeated lines from this set of strings -- you only need to check for `Invalid rate coefficient for reaction '<reaction string>'` once for each reaction.